cmake/Jemalloc.cmake

```diff
@@ -45,4 +51,17 @@ mark_as_advanced(
     JEMALLOC_LIBRARIES
     JEMALLOC_INCLUDE_DIR
 )
+if(JeMalloc_FOUND AND NOT (TARGET JeMalloc::JeMalloc))
```

Comment:
minor: maybe a comment to explain the places where this is not the case ?

Reply:
Just copied it from https://fossies.org/linux/ceph/cmake/modules/FindJeMalloc.cmake
Not sure that the `if(NOT TARGET` is necessary.

Reply:
I removed the second part of the if.